site/en/blog/chrome-120-beta-whats-new-for-extensions/index.md

```diff
@@ -0,0 +1,113 @@
+
+
+
+
+Our new [userScript sample](LINK) demonstrates a simple approach for detecting whether Developer mode is enabled and providing a simple onboarding flow.
+
+
+
+<figure>
+{% Img src="image/6hHqS5auVgWhN0cQNQztaJx5w4M2/Fh4eS4ovF8QboLiwLhY4.png", alt="Sample onboarding flow for user scripts.", width="800", height="401" %}
+<figcaption>Onboard users when Developer mode is disabled</figcaption>
+</figure>
+
+To get started check out the [documentation](LINK) or take a look at the [official sample](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/api-samples/userScripts).
+
+## Increased DNR limits
+
+Declarative Net Request (DNR) got two big updates this Chrome release. 
+
+### Higher static DNR ruleset limits
+
+The first update is big: higher static DNR ruleset limits.  We significantly increased the limit on enabled static rulesets from 10 to 50. Additionally, we increased the total number of allowed static rulesets from 50 to 100. This is in response to feedback we received in the Web Extensions Community Group.
+
+###  Higher dynamic rule limits
+
+The second update is even bigger: we are introducing the concept of safe rules in DNR, which allows extensions to increase the limit on the number of rules that can be added at runtime.
+
+It helps to first understand dynamic rules in general. The [`declarativeNetRequest`](/docs/extensions/reference/declarativeNetRequest/) API allows extensions to define rules which block or modify requests. Extensions can bundle rules with their extension, or they can add up to 5000 rules at runtime. One of the reasons for limiting the number of rules added at runtime is to mitigate the risks of unreviewed rules acting on requests, as some DNR rules can potentially be abused. Redirects, for example, can be used for phishing or affiliate link fraud. The risk is even higher when rules are loaded dynamically at runtime without being reviewed by the Web Store team.
+
+But it turned out that the initial limit of 5000 dynamic rules was too low. AdGuard reported that more than 2600 changes are made to popular lists each week, and of the 5% of users using custom filter lists, one in four of those users have a combined total of more than 5,000 custom rules across them (source). Adguard noted this as a significant challenge for migrating their extension to MV3 and we heard similar feedback from other ad blockers.
+
+We determined that some DNR rules such as "block" or "allow" are much safer and are less likely to be abused. The good news is, these rules make up the large majority of adblock filter rules, for example, 98-99% of all AdGuard’s filter rules are block rules. By raising the limit for dynamic rules only for these safe rules, we can provide more flexibility for ad blockers, while still keeping the same security guarantees as before - and this is what we’re doing in Chrome 120.
```

Comment:
```suggestion
We determined that some DNR rules such as "block" or "allow" are much safer and are less likely to be abused. The good news is, these rules make up the large majority of adblock filter rules, for example, 98 to 99 percent of all AdGuard’s filter rules are block rules. By raising the limit for dynamic rules only for safe rules, we can provide more flexibility for ad blockers, while still keeping the same security guarantees as before - and this is what we’re doing in Chrome 120.
```

Reply:
>large majority

This comment isn't blocking. I was going to comment on this in the Docs draft, but didn't have a suggestion and I still don't. 'Vast majority' is the more typical way of saying this in English, but I'm not sure that's what you're going for. I'm guessing 'majority' doesn't work because you're trying to convey something considerably larger than 50.01 percent, which is technically a majority, if not an impressive one.

Reply:
went with vast